images/makeproject/plan_class_spec.xml

```diff
@@ -10,8 +10,8 @@
         <opencl/>
     </plan_class>
     <plan_class>
-        <name>opencl_intel</name>
-        <gpu_type>intel</gpu_type>
+        <name>opencl_intel_gpu</name>
```

Comment:
Let's leave this as `opencl_intel`, the name is just cosmetic and this would avoid having to cut a new application version.